### DISCUSSION TITLE
managed to make a custom voice but cant use it

### DISCUSSION THREAD

**Discussion by usernamex1x23** [serp-ai/bark-with-voice-clone#17] [Q&A]
error i am get using google colab 

**I manage to clone my voice named the file first.npz**
then i changed the histroy_prompt to the name of the file 
audio_array = generate_audio(text_prompt, history_prompt=**"first"**)
Audio(audio_array, rate=SAMPLE_RATE)

**got this error any way to fix it** 

_AssertionError                            Traceback (most recent call last)
[<ipython-input-5-53eed1630342>](https://localhost:8080/#) in <cell line: 4>()
      2     [Excited]basic youtube video opening.  [Pause] sub to my channel,
      3 """
----> 4 audio_array = generate_audio(text_prompt, history_prompt="first")
      5 Audio(audio_array, rate=SAMPLE_RATE)

2 frames
[/usr/local/lib/python3.10/dist-packages/bark/generation.py](https://localhost:8080/#) in generate_text_semantic(text, history_prompt, temp, top_k, top_p, silent, min_eos_p, max_gen_duration_s, allow_early_stop, use_kv_caching)
    409             semantic_history = np.load(history_prompt)["semantic_prompt"]
    410         else:
--> 411             assert (history_prompt in ALLOWED_PROMPTS)
    412             semantic_history = np.load(
    413                 os.path.join(CUR_PATH, "assets", "prompts", f"{history_prompt}.npz")_

**Comment by devinschumacher** [comment]
@usernamex1x23  We just launched our follow up to BARK -- an AI Voice Cloner Desktop App (runs locally, offline, super realistic voice cloning, data privacy, no coding needed)

👉Check it out: https://www.youtube.com/watch?v=84JVGtkapik
